drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -64,6 +64,18 @@
                 <h3>Items</h3>
             </header>
             <section class="tab-content">
+                <section class="tab-content item-background-color-form">
+                    <label>
+                        Background color for items:
+                        <input class="item-background-color" type="text" value="{{ self.item_background_color}}">
+                    </label>
+                </section>
+                <section class="tab-content item-text-color-form">
+                    <label>
+                        Text color for items:
+                        <input class="item-text-color" type="text" value="{{ self.item_text_color}}">
```

Comment:
@bradenmacdonald OK, the input fields look like this now:

![input-fields-updated](https://cloud.githubusercontent.com/assets/961441/11557756/b61c83ca-99ae-11e5-8028-97f41a79da92.png)

I removed the single-quotes around `red` and `#ff0000` to avoid potential confusion about whether or not color names should be quoted.